src/app/models/types/translations.ts

```diff
@@ -116,6 +120,8 @@ export interface Translations {
     };
   };
   media: {
+    signPost?: string;
```

Comment:
Should we rename this to something like `sustainabilitySignpost` or something along those lines? (Or remove signpost from the name?)